examples/egress/egress-policy.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: "projectcalico.org/v1"
+kind: Egresspolicy
```

Comment:
OK - let's move ahead using the Calico spec / schema for a Policy, represented in a TPR.

One interesting point here is that Calico "selectors" are not intrinsically namespaced the way Kubernetes selectors are, meaning that a Calico Policy can select Pods across multiple / all namespaces.  We'll have to think about how this interacts with the fact that TPRs are currently Namespace scoped.

I think it's probably fine for now.  Note that in [this issue](https://github.com/kubernetes/features/issues/95) it looks like cluster-scoped TPRs are scheduled for Kubernetes v1.7, so when that rolls out we may want to revisit and make these cluster-scoped, but needs more thinking.

In the interest of keeping this moving (which I think is important), let's just mark the TPR apiVersion as `alpha`, indicating that there may be significant changes to this API in the future!